test/units/testsuite-58.sh

```diff
@@ -132,13 +132,15 @@ EOF
 Type=home
 Label=home-first
 Label=home-always-too-long-xxxxxxxxxxxxxx-%v
+Oem=yes
```

Comment:
`Oem` -> `OEM`

Reply:
Why the test succeeds??

Reply:
Improved the test so that it fails if OEM= is not enabled in this partition